dspace/config/modules/authentication-orcid.cfg

```diff
@@ -0,0 +1,30 @@
+#------------------------------------------------------------------#
+#--------------------ORCID GENERIC CONFIGURATIONS------------------#
+#------------------------------------------------------------------#
+
+#Allowed values are disabled, only_admin, only_owner or admin_and_owner
+orcid.disconnection.allowed-users = admin_and_owner
```

Comment:
While I understand the purpose of this configuration, I just realized that **because the ORCID tokens are stored in hidden metadata fields**, it is easy for an Administrator to bypass this setting, even if it is set to `only_owner`.  

An Admin can bypass this by simply deleting the ORCID tokens from metadata fields. Without those tokens, the synchronization is disconnected.

So, I'm not sure `only_owner` or `disabled` settings are possible. An Admin can always disconnect ORCID by editing the Item.